src/test/java/org/sagebionetworks/bridge/sdk/integration/StudyConsentTest.java

```diff
@@ -6,80 +6,91 @@
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.sagebionetworks.bridge.sdk.ResearcherClient;
+
+import org.sagebionetworks.bridge.Tests;
+import org.sagebionetworks.bridge.sdk.DeveloperClient;
 import org.sagebionetworks.bridge.sdk.Roles;
 import org.sagebionetworks.bridge.sdk.TestUserHelper;
 import org.sagebionetworks.bridge.sdk.TestUserHelper.TestUser;
 import org.sagebionetworks.bridge.sdk.exceptions.BridgeSDKException;
 import org.sagebionetworks.bridge.sdk.models.ResourceList;
+import org.sagebionetworks.bridge.sdk.models.holders.GuidVersionHolder;
 import org.sagebionetworks.bridge.sdk.models.subpopulations.StudyConsent;
+import org.sagebionetworks.bridge.sdk.models.subpopulations.Subpopulation;
+import org.sagebionetworks.bridge.sdk.models.subpopulations.SubpopulationGuid;
 
 public class StudyConsentTest {
 
-    private TestUser researcher;
+    private TestUser admin;
+    private TestUser developer;
+    private SubpopulationGuid subpopGuid;
 
     @Before
     public void before() {
-        researcher = TestUserHelper.createAndSignInUser(StudyConsentTest.class, true, Roles.RESEARCHER);
+        admin = TestUserHelper.getSignedInAdmin();
+        developer = TestUserHelper.createAndSignInUser(StudyConsentTest.class, true, Roles.DEVELOPER);
     }
 
     @After
     public void after() {
-        researcher.signOutAndDeleteUser();
+        if (subpopGuid != null) {
+            admin.getSession().getAdminClient().deleteSubpopulationPermanently(subpopGuid);    
+        }
+        developer.signOutAndDeleteUser();
     }
 
     @Test(expected=BridgeSDKException.class)
-    public void mustBeResearcherToAdd() {
+    public void mustBeDeveloperToAdd() {
```

Comment:
The user created in this test is not assigned the developer role. So although they retrieve the developer client and make the call, it should fail and throw a BridgeSDKException (it does). The next test, addAndActivateConsent(), requires a developer to work, so it seemed that it was de facto testing that the API was accessible by a developer.